var/spack/repos/builtin/packages/workrave/ax_cxx_compile_stdcxx.m4

```diff
@@ -0,0 +1,980 @@
+# ===========================================================================
+#  https://www.gnu.org/software/autoconf-archive/ax_cxx_compile_stdcxx.html
```

Comment:
@mmpozulp This looks to be available at http://git.savannah.gnu.org/gitweb/?p=autoconf-archive.git;a=blob_plain;f=m4/ax_cxx_compile_stdcxx.m4;hb=579023fd0623be745c643faca62c95adab584325 as part of a git repository`git://git.savannah.gnu.org/autoconf-archive.git`. Would you be able to manage this as a resource?

Reply:
Done. Updating my PR asap.